ZupIT/ritchie-cli
# refactor: deprecated dynamic input

### Description
In the coming months, due to the formula output feature, the type of dynamic input will be depreciated. 

Because of this, this PR adds a warning for users who use this type of input in their formulas, as shown in the GIF below:


<p align="center">
  <a rel="noopener" target="_blank"><img width="600px" src="https://media.giphy.com/media/D5y19E5qLw7HBm3Lm3/giphy.gif" alt="gif containing the command demonstration"></a>
</p>

Close #813 

### How to verify it
1. clone this PR (**`git fetch origin pull / 848 / head: feature/deprecated-dynamic-input`**)
2. create a formula with **dynamic input**
3. run the formula and check that the warning is displayed

### Changelog
Adds depreciation warning of dynamic input
